docs/OnTrack/Jupyter Notebook/Docker-Containers-SRS.md

```diff
@@ -0,0 +1,138 @@
+# Jupyter Notebook/Word Document Docker Containers Software Requirement Specification (SRS) Document
+
+---
```

Comment:
**suggestion:** I wonder if we can remove all the horizontal bars (ie. `---`) in this doc. The layout looks a bit odd to me.

<img width="441" alt="Screen Shot 2022-05-30 at 5 57 31 pm" src="https://user-images.githubusercontent.com/756722/170945455-55063d5b-9850-4260-873a-783309065a10.png">